core/condition.go

```diff
@@ -282,6 +282,15 @@ func prettify(parameters []string, resolvedParameters []string, operator string)
 	if strings.HasSuffix(resolvedParameters[0], InvalidConditionElementSuffix) || strings.HasSuffix(resolvedParameters[1], InvalidConditionElementSuffix) {
 		return resolvedParameters[0] + " " + operator + " " + resolvedParameters[1]
 	}
+
+	// If using pattern comparison, omit the contents
+	if strings.Contains(parameters[0], PatternFunctionPrefix) && len(resolvedParameters[1]) > 25 {
+		resolvedParameters[1] = fmt.Sprintf("%.25s...(truncated)", resolvedParameters[1])
+	}
+	if strings.Contains(parameters[1], PatternFunctionPrefix) && len(resolvedParameters[0]) > 25 {
```

Comment:
Instead of using `strings.Contains` to detect `PatternFunctionPrefix`, can you use `strings.HasPrefix(parameters[0], PatternFunctionPrefix) && strings.HasSuffix(first, FunctionSuffix)`?

It's very explicit, but it's just to prevent surprises.